Suggest  springcloud openjdk parent image for building task apps (#240)
spring-io/dataflow.spring.io: content/documentation/pages/5-feature-guides/3-batch/4-monitoring.md

</dependency>
```

<!--TIP-->

> For building a Task Docker image it is recommended to extend from the `springcloud/openjdk:latest` image. E.g. your task `Dockerfile` should start like this:

Reviewer: ```suggestion
To build a Docker image, you could extend from the `springcloud/openjdk:latest` base-image. For example, your task `Dockerfile` could start like this:
```
